### PR TITLE
Fk 359

### PR DESCRIPTION
**Pull request by GitError404**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/favrskov_helper/favrskov_helper.install [hunk 1]
```diff
@@ -572,3 +572,34 @@ function favrskov_helper_update_7030() {
...
+
+/**
+ * Updates news and archive views.
+ */
+function favrskov_helper_update_7031() {
+  // Updates news view.
+  features_revert_module('views_news');
+
+  // updates news.
+  $actual_time = time();
```
Comment (temoor-gilmutdinov) [inline modules/custom/favrskov_helper/favrskov_helper.install]: What's the point in putting time to a variable, if it's used only once?

File: modules/custom/favrskov_helper/favrskov_helper.install [hunk 2]
```diff
@@ -572,3 +572,34 @@ function favrskov_helper_update_7030() {
...
 }
+
+/**
+ * Updates news and archive views.
+ */
+function favrskov_helper_update_7031() {
+  // Updates news view.
+  features_revert_module('views_news');
+
+  // updates news.
```
Comment (temoor-gilmutdinov) [inline modules/custom/favrskov_helper/favrskov_helper.install]: Should respect the code style requirements.

File: modules/custom/favrskov_helper/favrskov_helper.install [hunk 3]
```diff
@@ -572,3 +572,34 @@ function favrskov_helper_update_7030() {
...
+  $actual_time = time();
+  $query = db_select('node', 'n')
+    ->fields('n', array('nid'))
+    ->condition('type', 'news')
+    ->condition('created', $actual_time, '<=')
+    ->condition('status', 1)
+    ->execute();
+
+  while($record = $query->fetchAssoc()) {
+    $is_created = db_select('autoarch_state', 'autoarch')
```
Comment (temoor-gilmutdinov) [inline modules/custom/favrskov_helper/favrskov_helper.install]: Why not joining this table to query above?

**Pull request closed without merging**